### PR TITLE
make use of mockito-release-tools v0.7.5 and configure commit message

### PR DESCRIPTION
**Pull request by epeee**
basically, making use of https://github.com/mockito/mockito-release-tools/issues/92 before making use of it in mockito project

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -4,7 +4,7 @@ buildscript {
         maven { url "https://plugins.gradle.org/m2/" }
     }
     dependencies {
-        def toolsVer = "0.7.1"
+        def toolsVer = "0.7.5"
```
Comment (mockitoguy) [inline build.gradle:7]: Nice! Thank you.

File: gradle/release.gradle [hunk 2]
```diff
@@ -11,6 +11,9 @@ releasing {
     git.releasableBranchRegex = "master|release/.+" // matches 'master', 'release/2.x', 'release/3.x', etc.
     git.branch = System.getenv("TRAVIS_BRANCH")
 
+    def buildNo = System.getenv("TRAVIS_BUILD_NUMBER")
```
Comment (mockitoguy) [inline gradle/release.gradle:14]: Let's hold off with this just yet... I'm trying to figure out how we can default that in a clean way in the release tools.

**Review by mockitoguy — DISMISSED** [review]
Give me a day, I'll try to figure out how to deal with Travis :)

**Comment by mockitoguy** [comment]
Actually, let's merge this. I'd like this example to demonstrate verbose configuration where most/all options are configured explicitly.

Thank you!

File: gradle/release.gradle (hunk 2 above)
Comment (epeee) [reply gradle/release.gradle:14]: just saw your recent changes in mockito-release-tools (TravisPlugin and so on). I really like the new TravisPlugin and therefore get rid of this configuration :+1:

**Pull request merged**